edx_notifications/stores/sql/migrations/0001_initial.py

```diff
@@ -1,122 +1,146 @@
-            'created': ('model_utils.fields.AutoCreatedField', [], {'default': 'datetime.datetime.now'}),
-            'deliver_no_earlier_than': ('django.db.models.fields.DateTimeField', [], {'null': 'True'}),
-            'expires_at': ('django.db.models.fields.DateTimeField', [], {'null': 'True', 'db_index': 'True'}),
-            'expires_secs_after_read': ('django.db.models.fields.IntegerField', [], {'null': 'True'}),
-            'from_user_id': ('django.db.models.fields.IntegerField', [], {'null': 'True'}),
-            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'modified': ('model_utils.fields.AutoLastModifiedField', [], {'default': 'datetime.datetime.now'}),
-            'msg_type': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['edx_notifications.SQLNotificationType']"}),
-            'namespace': ('django.db.models.fields.CharField', [], {'max_length': '128', 'null': 'True', 'db_index': 'True'}),
-            'payload': ('django.db.models.fields.TextField', [], {}),
-            'priority': ('django.db.models.fields.IntegerField', [], {'default': '0'})
-        },
-        'edx_notifications.sqlnotificationtype': {
-            'Meta': {'object_name': 'SQLNotificationType', 'db_table': "'edx_notifications_notificationtype'"},
-            'name': ('django.db.models.fields.CharField', [], {'max_length': '255', 'primary_key': 'True'}),
-            'renderer': ('django.db.models.fields.CharField', [], {'max_length': '255'})
-        },
-        'edx_notifications.sqlusernotification': {
-            'Meta': {'ordering': "['-created']", 'unique_together': "(('user_id', 'msg'),)", 'object_name': 'SQLUserNotification', 'db_table': "'edx_notifications_usernotification'"},
-            'created': ('model_utils.fields.AutoCreatedField', [], {'default': 'datetime.datetime.now'}),
-            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
-            'modified': ('model_utils.fields.AutoLastModifiedField', [], {'default': 'datetime.datetime.now'}),
-            'msg': ('django.db.models.fields.related.ForeignKey', [], {'to': "orm['edx_notifications.SQLNotificationMessage']"}),
-            'read_at': ('django.db.models.fields.DateTimeField', [], {'null': 'True', 'db_index': 'True'}),
-            'user_context': ('django.db.models.fields.TextField', [], {'null': 'True'}),
-            'user_id': ('django.db.models.fields.IntegerField', [], {'db_index': 'True'})
-        },
-        'edx_notifications.sqlusernotificationpreferences': {
-            'Meta': {'object_name': 'SQLUserNotificationPreferences', 'db_table': "'edx_notifications_usernotificationpreferences'"},
-            'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'})
-        }
-    }
-
-    complete_apps = ['edx_notifications']
\ No newline at end of file
+from __future__ import unicode_literals
```

Comment:
I'm assuming it's okay to update the implementation of this initial migration, since the old version is no longer applicable?